### PR TITLE
Improve testing framework

### PR DESCRIPTION
**Pull request by Jesus89**
Related to: https://github.com/CartoDB/renderer-prototype/issues/158

- Move testing GeoJSON files to `test/common/sources`
- Implement `f-` and `x-` detection for test's folders
- Update testing documentation
- Remove acceptance tests from ci

### CODE REVIEW

File: test/common/util.js [hunk 1]
```diff
@@ -6,10 +6,23 @@ const exquisite = require('exquisite-sst');
...
 
 function loadFiles(directory) {
     testsDir = directory;
-    return glob.sync(path.join(directory, '**', testFile));
+    let files = [];
+    let fFiles = [];
+    const allFiles = glob.sync(path.join(directory, '**', testFile));
+    allFiles.forEach(function (file) {
+        const name = getName(file);
+        if (name.indexOf('/f-') !== -1) {
```
Comment (IagoLast) [inline test/common/util.js]: [startsWith](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/String/startsWith)

File: test/common/util.js [hunk 2]
```diff
@@ -6,10 +6,23 @@ const exquisite = require('exquisite-sst');
...
+            fFiles.push(file);
+        } else if (name.indexOf('/x-') === -1) {
```
Comment (IagoLast) [inline test/common/util.js]: [same](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/String/startsWith)

File: test/common/util.js (hunk 1 above)
Comment (IagoLast) [reply test/common/util.js]: or (includes)[https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/String/includes]

File: test/integration/render/collection-point/strokeColor/scenario.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
-/* global carto geojson */
+/* global carto sources */
```
Comment (IagoLast) [inline test/integration/render/collection-point/strokeColor/scenario.js]: instead duplicating this line in every file we can add a .eslint.js file at top level

File: test/common/util.js (hunk 1 above)
Comment (Jesus89) [reply test/common/util.js]: startsWith does not work. includes is a better option ;)

**Pull request merged**